pkg/containerd/cmd/service/system_init.go

```diff
@@ -71,6 +76,32 @@ func systemInit(args []string) {
 		os.Exit(1)
 	}
 
+	// remove (unlikely) old containerd socket
+	_ = os.Remove(*sock)
+
+	// start up containerd
+	cmd := exec.Command(*binary)
+	cmd.Stderr = os.Stderr
```

Comment:
It might not now but who knows what it will do in the future, this isn't the sort of thing someone is likely to consider updating when bumping the containerd commit.

What's the harm of propagating stdout?